updating-bazel.md

```diff
@@ -26,7 +25,7 @@ We recommend the following process for project migration:
    "migration-\<release\>" label on GitHub, for example "[migration-0.26](https://github.com/bazelbuild/bazel/issues?utf8=%E2%9C%93&q=label%3Amigration-0.26+)".
```

Comment:
As discussed on the other PR, this section sounds like a list of steps to follow manually. I don't think it provides a good user experience. Maybe mention Bazelisk first in the page instead?

I think the most natural place to check when upgrading is the release announcement. We can mention the GitHub search on top of it, but I don't think most users need to do a GitHub search.

Reply:
Well I disagree with that, and I listed the reasons on the other PR (https://github.com/bazelbuild/bazel-website/pull/196) Feel free to continue the discussion there, but I am adressing your other comments with this PR.